sharepoint/sharepoint-ps/sharepoint-online/Invoke-SPOSiteSwap.md

```diff
@@ -47,7 +47,10 @@ If the target is the root site at <https://tenant-name.sharepoint.com,> then the
 3. Larger tenants that have more than ~10,000 licenses may also need to run the [Page Diagnostic Tool](https://docs.microsoft.com/office365/enterprise/page-diagnostics-for-spo) against the source site. Any analysis results that have the category Attention required (Red) or Improvement opprtunities (Orange) will need to be remediated before performing the swap.
 
 The source and target sites can't be connected to an Office 365 group. They also can't be hub sites or associated with a hub.
-If a site is a hub site, unregister it as a hub site, swap the root site, and then register the site as a hub site. If a site is associated with a hub, disassociate the site, swap the root site, and then reassociate the site.
+If a site is a hub site, unregister it as a hub site, swap the root site, and then register the site as a hub site. If a site is associated with a hub, disassociate the site, swap the root site, and then reassociate the site. 
+
+> [!IMPORTANT]
```

Comment:
Tags such as !IMPORTANT should not be used in cmdlet documentation as these do not render in PowerShell.

Reply:
Ok